text/0001-rfc-format.md

```diff
@@ -0,0 +1,32 @@
+  * Start date: 2018-02-21
+  * Contributors: Lorenz Meier <lorenz@px4.io>, ...
+  * Related issues: 
+  
+# Summary
+
+The scope of this document is to design a Request-for-comments process for MAVLink for protocol additions that require a non-trivial change. The goal is to create a transparent, agreed-on process that allows the community and maintainer to make informed decisions and to capture the rationales behind these decisions in a document. This will insure quality of the design as well as later inform developers who have not been part of the original decisions of the reasons. The documents should also include further references and examples.
+  
+# Motivation
+
+As the amount of micro-services in the MAVLink protocol increases the design process needs more structure to be able to accomodate all stakeholders and create the required transparency.
+
+# Detailed Design
+
+The RFC process should follow a simple flow:
+
+  * The original proposer raises a pull request with a document in this format.
```

Comment:
You mean the format of this document? So "The RFC proposer raises a pull request with the same format/headings as this proposal."